nautobot/ipam/api/views.py

```diff
@@ -356,7 +356,7 @@ class PrefixLocationAssignmentViewSet(NautobotModelViewSet):
 
 
 class IPAddressViewSet(NautobotModelViewSet):
-    queryset = IPAddress.objects.all()
+    queryset = IPAddress.objects.select_related("parent__namespace")
```

Comment:
We don't auto-optimize nested lookups yet, and we weren't correctly selecting this one before. Another separate query I noticed testing this endpoint.